packages/flutter_localizations/lib/src/l10n/material_en.arb

```diff
@@ -207,7 +207,7 @@
     "description": "The label for the Search Web button and menu items on iOS."
   },
 
-  "shareButtonLabel": "Share...",
+  "shareButtonLabel": "Share",
```

Comment:
I may be misunderstanding this but according to this PR's description, `This PR updates the Material label for the share option, currently it is "Share..." (the expected label on iOS), on Android it should be "Share".` So it seems like this value shouldn't change?

Reply:
Sorry if this PR description is confusing, this change is the purpose of this PR: displaying 'Share' instead of 'Share...'.
'Share' is the native Android label, 'Share...' is the native iOS label.

This is similar to the select all label which is 'Select all' in Material localizations (same as native Android) and 'Select All' in Cupertino localizations (same as native iOS) :

Select all label for Material:
https://github.com/flutter/flutter/blob/89edac404184db1dc9a04b4bad615e28d946037b/packages/flutter/lib/src/material/material_localizations.dart#L1189

Select all label for Cupertino:
https://github.com/flutter/flutter/blob/89edac404184db1dc9a04b4bad615e28d946037b/packages/flutter/lib/src/cupertino/localizations.dart#L486

Reply:
Thanks for the explanation. It was the `description` of the shareButtonLabel message that threw me off; looks like you've already fixed that.